pkg/proc/proc_test.go

```diff
@@ -3521,6 +3584,9 @@ func TestIssue1145(t *testing.T) {
 }
 
 func TestDisassembleGlobalVars(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
If I'm not mistaken the disasm has been implemented. Is there any other reason to skip this test?

Reply:
Mainly may be testsuit's bug, It seems can't find main.v in range of mainfn.Entry and mainfn.End on arm64. Needs to investigate in #1770.